libs/blocks/review/review.js

```diff
@@ -121,6 +121,11 @@ export default async function init(el) {
   loadStyle(`${base}/ui/page/page.css`);
   const metaData = getMetaData(el);
   const strings = getStrings(metaData);
+
+  if (window.location.host === 'www.stage.adobe.com') {
```

Comment:
@narcis-radu @mokimo @draganatrajkovic 
would this make everyone happy?
```
  if (window.location.host != 'www.adobe.com') { ... }

Reply:
If `stage.adobe.com` and `adobe.com` is enough sure. I'm don't have the context whether that works for all use-cases/consumers.

Who's the code-owner for the review block anyway? @nkthakur48 Do you know?

Reply:
@mokimo Could we explore other approaches (like specifying an optional stage submission URL through authoring; or swapping domain if Milo somewhere has config details for all consuming domains, stage and prod)? Personally I think harcoding checks for domain name will reduce the re-usability of this component

CC: @Blainegunn @mokimo

Reply:
@nkthakur48 @mokimo @Blainegunn 
Can you please check my fix?